app/controllers/settings_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
 
   def update
     @settings = current_user
-    if @settings.update_attributes(settings_params)
+    if settings_params[:email] =~ /^.+@.+$/ && @settings.update_attributes(settings_params)
```

Comment:
We're going to devote 5 minutes of class during the "How to email" week on why regex is bad for email :grinning: :wink: